tests/unit/split_process_line_test.c

```diff
@@ -117,6 +117,42 @@ static void test_split_line_noelapsed(void)
     free_and_null_strings(field);
 }
 
+static void test_split_line_nocmd(void)
+{
+    /* Collect all test data in one array to make alignments visible: */
+    static const char *lines[] = {
+        /* Indent any continuation lines so that it's clear that's what they are. */
+        /* Use the username field as a test name to confirm tests are in sync. */
+
+        "USER       PID STAT    VSZ  NI   RSS NLWP STIME     ELAPSED     TIME COMMAND",
```

Comment:
If this is exactly copied from a (ps) command, putting the command in a comment here would be helpful.

If not, is it intentional / important that the labels and values are sometimes misaligned? It could help readability if you added some spaces to align things perfectly.